source/Purchases.brs

```diff
@@ -953,9 +953,9 @@ function _InternalPurchases(o = {} as object) as object
         end function
         willRenew: function(purchase) as boolean
             isPromo = purchase.store = "promotional"
-            isLifetime = purchase.expirationDate = invalid
-            hasUnsubscribed = purchase.unsubscribeDetectedAt <> invalid
-            hasBillingIssues = purchase.billingIssueDetectedAt <> invalid
+            isLifetime = purchase.expires_date = invalid
+            hasUnsubscribed = purchase.unsubscribe_detected_at <> invalid
+            hasBillingIssues = purchase.billing_issue_detected_at <> invalid
```

Comment:
We were using the already-parsed field names, but the parameter contains the raw purchase object